src/pages/julieLee/Login/Login.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import "./Login.scss";
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+
+function LoginJulie() {
+  const navigate = useNavigate();
+  const goToMain = () => {
+    navigate("/main-julie");
+  };
+
+  const [identify, setIdentify] = useState("");
+  const handleIdInput = (event) => setIdentify(event.target.value);
+  const [password, setPassword] = useState("");
+  const handlePwInput = (event) => setPassword(event.target.value);
+
+  const validation = (idText, pwText) => {
+    if (idText.includes("@") && pwText.length >= 5) {
+      return true;
+    }
+  };
+
+  const runValidation = validation(identify, password);
+
+  const [opacity, setOpacity] = useState(0.5);
+  const handleBtn = () => {
+    runValidation ? setOpacity(1) : setOpacity(0.5);
+  };
+  return (
+    <div>
+      <div className="container">
+        <div className="logo-box">
+          <span className="logo-text">Justgram</span>
+        </div>
```

Comment:
```suggestion
      <div className={styles.container}>
        <div className={styles.logo-box}>
          <span className={styles.logo-text}>Justgram</span>
        </div>
```

module.scss 사용 예시입니다.